pkg/network/usm/usm_http2_monitor_test.go

```diff
@@ -518,6 +519,9 @@ func (s *usmHTTP2Suite) TestRawTraffic() {
 					Method: usmhttp.MethodPost,
 				}: 1,
 			},
+			// Currently we don't have a way to test it as TLS version does not have the ability to run 2 tail calls
+			// for filtering frames (will be fixed in USMON-684)
+			skip: s.isTLS,
```

Comment:
why?

Reply:
Mistake from the merge, thanks for the catch!